lapce-data/src/data.rs

```diff
@@ -228,12 +237,37 @@ impl LapceData {
         }
 
         if let Some((window_id, _)) = windows.iter().next() {
-            for file in files {
+            for (path, line_col) in parsed_paths {
+                let widget_id = Arc::try_unwrap(
+                    windows.get(window_id).unwrap().active_id.clone(),
+                )
+                .unwrap();
+
+                // open the file
                 let _ = event_sink.submit_command(
                     LAPCE_UI_COMMAND,
-                    LapceUICommand::OpenFile(file.to_path_buf(), false),
+                    LapceUICommand::OpenFile(path.clone(), false),
                     Target::Window(*window_id),
                 );
+
+                // jump to line and column
+                let _ = event_sink.submit_command(
+                    LAPCE_UI_COMMAND,
+                    LapceUICommand::JumpToLineColLocation(
+                        Some(widget_id),
+                        EditorLocation {
+                            path: path.clone(),
+                            position: Some(line_col),
+                            scroll_offset: Some(Vec2 {
+                                x: line_col.column as f64,
+                                y: line_col.line as f64,
+                            }),
+                            history: None,
+                        },
+                        true,
+                    ),
+                    Target::Widget(widget_id),
+                );
             }
```

Comment:
Copy this part from @panekj work: https://github.com/panekj/lapce/commit/e9e091f34ee48440145f5b8e76662b9666eeebca#diff-70e9249c3d8d41c2edb4e1190bdec53808ae9fb1b2fda9a0ebd2c2ed45d5ebffR303-R328